### PR TITLE
Refresh Fixity Documentation

### PR DESCRIPTION
**Pull request by ross-spencer**
This PR refreshes the Fixity documentation.

For reviewers, you'll see 8 commits which I've tried to group as logically as possible for review. I propose to squish them all into a single commit before merging. 

Of the four files modified, two are generated automatically using https://github.com/rtomayko/ronn from [fixity.1.md](https://github.com/artefactual/fixity/blob/dev/issue-317-refresh-fixity-docs/docs/fixity.1.md)

The two that have been updated for their content, aside from their diffs, are best viewed in a GitHub markdown viewer:

* README.md: https://github.com/artefactual/fixity/blob/dev/issue-317-refresh-fixity-docs/README.md
* Manual Page: https://github.com/artefactual/fixity/blob/dev/issue-317-refresh-fixity-docs/docs/fixity.1.md

Connected to archivematica/issues#317

### CODE REVIEW

**Review by ablwr — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,15 +1,240 @@
 fixity
 ======
 
-Fixity is a simple commandline tool that assists in checking fixity for AIPs stored in Archivematica Storage Service instances.
+Fixity is a command line tool that assists in checking fixity for AIPs stored
```
Comment (sallain) [inline README.md:4]: Defining fixity very briefly here might be helpful - even just a link out to a resource like the [DPC page on fixity](https://www.dpconline.org/handbook/technical-solutions-and-tools/fixity-and-checksums)

**Review by sallain — APPROVED** [review]
LGTM! I made one small recommendation but it is optional.

File: README.md (hunk 1 above)
Comment (ross-spencer) [reply README.md:4]: Nice idea. I've added that now. Thanks @sallain :+1:

Comment (sallain) [reply README.md:4]: :1st_place_medal:

**Pull request merged**